packages/teams-js/src/public/constants.ts

```diff
@@ -1,21 +1,33 @@
+/** HostClientType represents the different client platforms on which host can be run. */
 export enum HostClientType {
+  /** Represents the desktop client of host, which is installed on a user's computer and runs as a standalone application. */
   desktop = 'desktop',
+  /** Represents the web-based client of host, which runs in a web browser. */
   web = 'web',
+  /** Represents the Android mobile client of host, which runs on Android devices such as smartphones and tablets. */
   android = 'android',
+  /** Represents the iOS mobile client of host, which runs on iOS devices such as iPhones. */
   ios = 'ios',
+  /** Represents the iPadOS client of host, which runs on iOS devices such as iPads. */
   ipados = 'ipados',
   /**
    * @deprecated
    * As of 2.0.0, please use {@link teamsRoomsWindows} instead.
    */
   rigel = 'rigel',
+  /** Represents the client of host, which runs on surface hub devices. */
   surfaceHub = 'surfaceHub',
+  /** Represents the client of host, which runs on Teams Rooms on Windows devices. More information on Microsoft Teams Rooms on Windows can be found [here](https://support.microsoft.com/en-us/office/microsoft-teams-rooms-windows-help-e667f40e-5aab-40c1-bd68-611fe0002ba2)*/
   teamsRoomsWindows = 'teamsRoomsWindows',
+  /** Represents the client of host, which runs on Teams Rooms on Android devices. More information on Microsoft Teams Rooms on Android can be found [here].(https://support.microsoft.com/en-us/office/get-started-with-teams-rooms-on-android-68517298-d513-46be-8d6d-d41db5e6b4b2)*/
   teamsRoomsAndroid = 'teamsRoomsAndroid',
+  /** Represents the client of host, which runs on Teams phones. More information can be found [here](https://support.microsoft.com/en-us/office/get-started-with-teams-phones-694ca17d-3ecf-40ca-b45e-d21b2c442412) */
   teamsPhones = 'teamsPhones',
+  /** Represents the client of host, which runs on Teams displays devices. More information can be found [here](https://support.microsoft.com/en-us/office/get-started-with-teams-displays-ff299825-7f13-4528-96c2-1d3437e6d4e6) */
   teamsDisplays = 'teamsDisplays',
 }
 
+/** HostName indicates the host under which the application is presently executing. */
```

Comment:
>the application [](http://example.com/codeflow?start=44&length=15)

To make it clearer which "application" this refers to, consider rephasing to say "your app," e.g.,

"Possible hosts for your app"

Reply:
Sorry, I'm bit confused here, do you mean something like this?

`HostName indicates the possible hosts for your application`

Reply:
Yes!

Reply:
Will update in latest commit.